src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageDependencies.cs

```diff
@@ -163,6 +176,55 @@ protected override void ExecuteCore()
             ReadProjectFileDependencies();
             RaiseLockFileTargets();
             GetPackageAndFileDefinitions();
+
+            FixUpNetStandardLibraryFacadeAssemblies();
+        }
+
+        /// <remarks>
+        /// The NETStandard.Library package (starting from version 2.0) adds references
+        /// to most assemblies via a .targets file rather than going through the normal
+        /// process to resolve assemblies from packages. We need to find these and
+        /// create the appropriate file definitions and dependencies to associate them
+        /// with the package.
+        /// Here we find the "facade" references from the .NETStandard.Library package,
+        /// update the existing FileDefinition, and create a FileDependency between the
```

Comment:
Different parts of the tree use different items. The nodes shown under Packages and SDK are based in part on the `FileDefinition` and `FileDependency` items.